json/src/main/java/org/openx/data/jsonserde/util/ThreadLocalSimpleDateFormat.java

```diff
@@ -0,0 +1,52 @@
+package org.openx.data.jsonserde.util;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+/**
+ * Wraps a {@link SimpleDateFormat} instance with a {@link ThreadLocal} since the former is not thread safe and
+ * mutates more than a few underlying fields directly during parsing. When used from multiple fields, this can
+ * corrupt data in a way that may or may not cause exceptions (ie: can cause silent corruption).
+ */
+public final class ThreadLocalSimpleDateFormat {
```

Comment:
package private, so it doesn't become considered API

Reply:
Moved packages and made package private.